web/modules/custom/dpl_something_similar/src/Plugin/Block/SomethingSimilarBlock.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Drupal\dpl_something_similar\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\dpl_react_apps\Controller\DplReactAppsController;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a "something similar" component.
+ *
+ * @Block(
+ *   id = "dpl_something_similar_block",
+ *   admin_label = "Something similar"
+ * )
+ */
+class SomethingSimilarBlock extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Drupal config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  private ConfigFactoryInterface $configFactory;
+
+  /**
+   * SomethingSimilarBlock constructor.
+   *
+   * @param mixed[] $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin ID for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
+   *   Drupal config factory to get FBS and Publizon settings.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, ConfigFactoryInterface $configFactory) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->configuration = $configuration;
+    $this->configFactory = $configFactory;
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   *   The service container.
+   * @param mixed[] $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin ID for the plugin instance.
+   * @param int $plugin_definition
+   *   The plugin implementation definition.
+   *
+   * @return \Drupal\dpl_something_similar\Plugin\Block\SomethingSimilarBlock|static
+   *   Something similar.
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('config.factory'),
+    );
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * @return mixed[]
+   *   The app render array.
+   */
+  public function build() {
+    $fbsConfig = $this->configFactory->get('dpl_fbs.settings');
+    $publizonConfig = $this->configFactory->get('dpl_publizon.settings');
+    $data = [
+      // Urls.
+      "fbs-base-url" => $fbsConfig->get('base_url'),
+      "publizon-base-url" => $publizonConfig->get('base_url'),
```

Comment:
These base url's are also added by `DplReactAppsController::externalApiBaseUrls()` in line 93

Reply:
removed